index.html

```diff
@@ -8,6 +8,7 @@
     <link rel="stylesheet" href="style.css">
   </head>
   <body>
+  <div>This is my balls</div>
```

Comment:
this is dumb plz change. balls has a z
```suggestion
  <div>This is my ballzz</div>
```